Reposition deb packages

Currently the deb packages we use to install Chaste dependencies on Ubuntu are located here:
https://www.cs.ox.ac.uk/chaste/ubuntu/XYZ

This is currently causing problems as the web server is really struggling of late:
```
Resolving www.cs.ox.ac.uk (www.cs.ox.ac.uk)... 129.67.[15](https://github.com/Chaste/Chaste/actions/runs/5583476368/jobs/10203834017?pr=13#step:4:16)1.1
Connecting to www.cs.ox.ac.uk (www.cs.ox.ac.uk)|129.67.151.1|:443... connected.
OpenSSL: error:0A000152:SSL routines::unsafe legacy renegotiation disabled
Unable to establish SSL connection.
Error: Process completed with exit code 4.
```

and this is causing our CI runs to fall over.

The appropriate solution is to reposition these to the new website.  The proposal would be that they end up at:
https://chaste.github.io/ubuntu/XYZ

This can be achieved by creating a repository called `ubuntu` in Chaste organisation and allowing that to serve files using GitHub pages.

## Todo:

- [x] Create new repo
- [x] Reposition deb packages
- [x] Edit CI workflows to all use new location
- [x] Edit install guides to point to new location
- [ ] Edit developer guide on how to create packages for new Ubuntus
- [ ] Do a global search through all existing documentation to ensure no references to old location remains

--- a/.github/workflows/profiling-gprof.yml
+++ b/.github/workflows/profiling-gprof.yml
@@ -37,8 +37,8 @@ jobs:
 
       - name: install dependencies
         run: |
-          echo 'deb [signed-by=/usr/share/keyrings/chaste.asc] https://www.cs.ox.ac.uk/chaste/ubuntu jammy/' | sudo tee -a /etc/apt/sources.list.d/chaste.list
-          sudo wget -O /usr/share/keyrings/chaste.asc https://www.cs.ox.ac.uk/chaste/ubuntu/ChasteTeam.asc
+          echo 'deb [signed-by=/usr/share/keyrings/chaste.asc] https://chaste.github.io/ubuntu/ jammy/' | sudo tee -a /etc/apt/sources.list.d/chaste.list
+          sudo wget -O /usr/share/keyrings/chaste.asc https://chaste.github.io/ubuntu//ChasteTeam.asc
           sudo apt update
           sudo apt install chaste-dependencies
 

--- a/.github/workflows/profiling-gprof.yml
+++ b/.github/workflows/profiling-gprof.yml
@@ -38,7 +38,7 @@ jobs:
       - name: install dependencies
         run: |
           echo 'deb [signed-by=/usr/share/keyrings/chaste.asc] https://chaste.github.io/ubuntu jammy/' | sudo tee -a /etc/apt/sources.list.d/chaste.list
-          sudo wget -O /usr/share/keyrings/chaste.asc https://chaste.github.io/ubuntu/ChasteTeam.asc
+          sudo wget -O /usr/share/keyrings/chaste.asc https://chaste.github.io/chaste.asc
           sudo apt update
           sudo apt install chaste-dependencies
 

--- a/.github/workflows/profiling-gprof.yml
+++ b/.github/workflows/profiling-gprof.yml
@@ -37,8 +37,8 @@ jobs:
 
       - name: install dependencies
         run: |
-          echo 'deb [signed-by=/usr/share/keyrings/chaste.asc] https://chaste.github.io/ubuntu/ jammy/' | sudo tee -a /etc/apt/sources.list.d/chaste.list
-          sudo wget -O /usr/share/keyrings/chaste.asc https://chaste.github.io/ubuntu//ChasteTeam.asc
+          echo 'deb [signed-by=/usr/share/keyrings/chaste.asc] https://chaste.github.io/ubuntu jammy/' | sudo tee -a /etc/apt/sources.list.d/chaste.list
+          sudo wget -O /usr/share/keyrings/chaste.asc https://chaste.github.io/ubuntu/ChasteTeam.asc
           sudo apt update
           sudo apt install chaste-dependencies
 

--- a/.github/workflows/project-tests.yml
+++ b/.github/workflows/project-tests.yml
@@ -35,8 +35,8 @@ jobs:
 
     - name: install dependencies
       run: |
-        echo 'deb [signed-by=/usr/share/keyrings/chaste.asc] https://www.cs.ox.ac.uk/chaste/ubuntu jammy/' | sudo tee -a /etc/apt/sources.list.d/chaste.list
-        sudo wget -O /usr/share/keyrings/chaste.asc https://www.cs.ox.ac.uk/chaste/ubuntu/ChasteTeam.asc
+        echo 'deb [signed-by=/usr/share/keyrings/chaste.asc] https://chaste.github.io/ubuntu/ jammy/' | sudo tee -a /etc/apt/sources.list.d/chaste.list
+        sudo wget -O /usr/share/keyrings/chaste.asc https://chaste.github.io/ubuntu//ChasteTeam.asc
         sudo apt update
         sudo apt install chaste-dependencies
     

--- a/.github/workflows/project-tests.yml
+++ b/.github/workflows/project-tests.yml
@@ -36,7 +36,7 @@ jobs:
     - name: install dependencies
       run: |
         echo 'deb [signed-by=/usr/share/keyrings/chaste.asc] https://chaste.github.io/ubuntu jammy/' | sudo tee -a /etc/apt/sources.list.d/chaste.list
-        sudo wget -O /usr/share/keyrings/chaste.asc https://chaste.github.io/ubuntu/ChasteTeam.asc
+        sudo wget -O /usr/share/keyrings/chaste.asc https://chaste.github.io/chaste.asc
         sudo apt update
         sudo apt install chaste-dependencies
     

--- a/.github/workflows/project-tests.yml
+++ b/.github/workflows/project-tests.yml
@@ -35,8 +35,8 @@ jobs:
 
     - name: install dependencies
       run: |
-        echo 'deb [signed-by=/usr/share/keyrings/chaste.asc] https://chaste.github.io/ubuntu/ jammy/' | sudo tee -a /etc/apt/sources.list.d/chaste.list
-        sudo wget -O /usr/share/keyrings/chaste.asc https://chaste.github.io/ubuntu//ChasteTeam.asc
+        echo 'deb [signed-by=/usr/share/keyrings/chaste.asc] https://chaste.github.io/ubuntu jammy/' | sudo tee -a /etc/apt/sources.list.d/chaste.list
+        sudo wget -O /usr/share/keyrings/chaste.asc https://chaste.github.io/ubuntu/ChasteTeam.asc
         sudo apt update
         sudo apt install chaste-dependencies
     

--- a/.github/workflows/ubuntu-compilers.yml
+++ b/.github/workflows/ubuntu-compilers.yml
@@ -52,7 +52,7 @@ jobs:
     - name: install dependencies
       run: |
         echo 'deb [signed-by=/usr/share/keyrings/chaste.asc] https://chaste.github.io/ubuntu jammy/' | sudo tee -a /etc/apt/sources.list.d/chaste.list
-        sudo wget -O /usr/share/keyrings/chaste.asc https://chaste.github.io/ubuntu/ChasteTeam.asc
+        sudo wget -O /usr/share/keyrings/chaste.asc https://chaste.github.io/chaste.asc
         sudo apt update
         sudo apt install chaste-dependencies
 

--- a/.github/workflows/ubuntu-compilers.yml
+++ b/.github/workflows/ubuntu-compilers.yml
@@ -51,8 +51,8 @@ jobs:
 
     - name: install dependencies
       run: |
-        echo 'deb [signed-by=/usr/share/keyrings/chaste.asc] https://chaste.github.io/ubuntu/ jammy/' | sudo tee -a /etc/apt/sources.list.d/chaste.list
-        sudo wget -O /usr/share/keyrings/chaste.asc https://chaste.github.io/ubuntu//ChasteTeam.asc
+        echo 'deb [signed-by=/usr/share/keyrings/chaste.asc] https://chaste.github.io/ubuntu jammy/' | sudo tee -a /etc/apt/sources.list.d/chaste.list
+        sudo wget -O /usr/share/keyrings/chaste.asc https://chaste.github.io/ubuntu/ChasteTeam.asc
         sudo apt update
         sudo apt install chaste-dependencies
 

--- a/.github/workflows/ubuntu-compilers.yml
+++ b/.github/workflows/ubuntu-compilers.yml
@@ -51,8 +51,8 @@ jobs:
 
     - name: install dependencies
       run: |
-        echo 'deb [signed-by=/usr/share/keyrings/chaste.asc] https://www.cs.ox.ac.uk/chaste/ubuntu jammy/' | sudo tee -a /etc/apt/sources.list.d/chaste.list
-        sudo wget -O /usr/share/keyrings/chaste.asc https://www.cs.ox.ac.uk/chaste/ubuntu/ChasteTeam.asc
+        echo 'deb [signed-by=/usr/share/keyrings/chaste.asc] https://chaste.github.io/ubuntu/ jammy/' | sudo tee -a /etc/apt/sources.list.d/chaste.list
+        sudo wget -O /usr/share/keyrings/chaste.asc https://chaste.github.io/ubuntu//ChasteTeam.asc
         sudo apt update
         sudo apt install chaste-dependencies
 

--- a/.github/workflows/ubuntu-infrastructure.yml
+++ b/.github/workflows/ubuntu-infrastructure.yml
@@ -28,8 +28,8 @@ jobs:
 
     - name: install dependencies
       run: |
-        echo 'deb [signed-by=/usr/share/keyrings/chaste.asc] https://chaste.github.io/ubuntu/ focal/' | sudo tee -a /etc/apt/sources.list.d/chaste.list
-        sudo wget -O /usr/share/keyrings/chaste.asc https://chaste.github.io/ubuntu//ChasteTeam.asc
+        echo 'deb [signed-by=/usr/share/keyrings/chaste.asc] https://chaste.github.io/ubuntu focal/' | sudo tee -a /etc/apt/sources.list.d/chaste.list
+        sudo wget -O /usr/share/keyrings/chaste.asc https://chaste.github.io/ubuntu/ChasteTeam.asc
         sudo apt update
         sudo apt install chaste-dependencies
 

--- a/.github/workflows/ubuntu-infrastructure.yml
+++ b/.github/workflows/ubuntu-infrastructure.yml
@@ -29,7 +29,7 @@ jobs:
     - name: install dependencies
       run: |
         echo 'deb [signed-by=/usr/share/keyrings/chaste.asc] https://chaste.github.io/ubuntu focal/' | sudo tee -a /etc/apt/sources.list.d/chaste.list
-        sudo wget -O /usr/share/keyrings/chaste.asc https://chaste.github.io/ubuntu/ChasteTeam.asc
+        sudo wget -O /usr/share/keyrings/chaste.asc https://chaste.github.io/chaste.asc
         sudo apt update
         sudo apt install chaste-dependencies
 

--- a/.github/workflows/ubuntu-infrastructure.yml
+++ b/.github/workflows/ubuntu-infrastructure.yml
@@ -28,8 +28,8 @@ jobs:
 
     - name: install dependencies
       run: |
-        echo 'deb [signed-by=/usr/share/keyrings/chaste.asc] https://www.cs.ox.ac.uk/chaste/ubuntu focal/' | sudo tee -a /etc/apt/sources.list.d/chaste.list
-        sudo wget -O /usr/share/keyrings/chaste.asc https://www.cs.ox.ac.uk/chaste/ubuntu/ChasteTeam.asc
+        echo 'deb [signed-by=/usr/share/keyrings/chaste.asc] https://chaste.github.io/ubuntu/ focal/' | sudo tee -a /etc/apt/sources.list.d/chaste.list
+        sudo wget -O /usr/share/keyrings/chaste.asc https://chaste.github.io/ubuntu//ChasteTeam.asc
         sudo apt update
         sudo apt install chaste-dependencies
 

--- a/.github/workflows/ubuntu-parallel.yml
+++ b/.github/workflows/ubuntu-parallel.yml
@@ -37,8 +37,8 @@ jobs:
 
     - name: install dependencies
       run: |
-        echo 'deb [signed-by=/usr/share/keyrings/chaste.asc] https://chaste.github.io/ubuntu/ focal/' | sudo tee -a /etc/apt/sources.list.d/chaste.list
-        sudo wget -O /usr/share/keyrings/chaste.asc https://chaste.github.io/ubuntu//ChasteTeam.asc
+        echo 'deb [signed-by=/usr/share/keyrings/chaste.asc] https://chaste.github.io/ubuntu focal/' | sudo tee -a /etc/apt/sources.list.d/chaste.list
+        sudo wget -O /usr/share/keyrings/chaste.asc https://chaste.github.io/ubuntu/ChasteTeam.asc
         sudo apt update
         sudo apt install chaste-dependencies
 

--- a/.github/workflows/ubuntu-parallel.yml
+++ b/.github/workflows/ubuntu-parallel.yml
@@ -38,7 +38,7 @@ jobs:
     - name: install dependencies
       run: |
         echo 'deb [signed-by=/usr/share/keyrings/chaste.asc] https://chaste.github.io/ubuntu focal/' | sudo tee -a /etc/apt/sources.list.d/chaste.list
-        sudo wget -O /usr/share/keyrings/chaste.asc https://chaste.github.io/ubuntu/ChasteTeam.asc
+        sudo wget -O /usr/share/keyrings/chaste.asc https://chaste.github.io/chaste.asc
         sudo apt update
         sudo apt install chaste-dependencies
 

--- a/.github/workflows/ubuntu-parallel.yml
+++ b/.github/workflows/ubuntu-parallel.yml
@@ -37,8 +37,8 @@ jobs:
 
     - name: install dependencies
       run: |
-        echo 'deb [signed-by=/usr/share/keyrings/chaste.asc] https://www.cs.ox.ac.uk/chaste/ubuntu focal/' | sudo tee -a /etc/apt/sources.list.d/chaste.list
-        sudo wget -O /usr/share/keyrings/chaste.asc https://www.cs.ox.ac.uk/chaste/ubuntu/ChasteTeam.asc
+        echo 'deb [signed-by=/usr/share/keyrings/chaste.asc] https://chaste.github.io/ubuntu/ focal/' | sudo tee -a /etc/apt/sources.list.d/chaste.list
+        sudo wget -O /usr/share/keyrings/chaste.asc https://chaste.github.io/ubuntu//ChasteTeam.asc
         sudo apt update
         sudo apt install chaste-dependencies
 

--- a/.github/workflows/ubuntu-tests.yml
+++ b/.github/workflows/ubuntu-tests.yml
@@ -45,8 +45,8 @@ jobs:
 
     - name: install dependencies
       run: |
-        echo 'deb [signed-by=/usr/share/keyrings/chaste.asc] https://www.cs.ox.ac.uk/chaste/ubuntu ${{ matrix.codename }}/' | sudo tee -a /etc/apt/sources.list.d/chaste.list
-        sudo wget -O /usr/share/keyrings/chaste.asc https://www.cs.ox.ac.uk/chaste/ubuntu/ChasteTeam.asc
+        echo 'deb [signed-by=/usr/share/keyrings/chaste.asc] https://chaste.github.io/ubuntu/ ${{ matrix.codename }}/' | sudo tee -a /etc/apt/sources.list.d/chaste.list
+        sudo wget -O /usr/share/keyrings/chaste.asc https://chaste.github.io/ubuntu//ChasteTeam.asc
         sudo apt update
         sudo apt install chaste-dependencies
 

--- a/.github/workflows/ubuntu-tests.yml
+++ b/.github/workflows/ubuntu-tests.yml
@@ -45,8 +45,8 @@ jobs:
 
     - name: install dependencies
       run: |
-        echo 'deb [signed-by=/usr/share/keyrings/chaste.asc] https://chaste.github.io/ubuntu/ ${{ matrix.codename }}/' | sudo tee -a /etc/apt/sources.list.d/chaste.list
-        sudo wget -O /usr/share/keyrings/chaste.asc https://chaste.github.io/ubuntu//ChasteTeam.asc
+        echo 'deb [signed-by=/usr/share/keyrings/chaste.asc] https://chaste.github.io/ubuntu ${{ matrix.codename }}/' | sudo tee -a /etc/apt/sources.list.d/chaste.list
+        sudo wget -O /usr/share/keyrings/chaste.asc https://chaste.github.io/ubuntu/ChasteTeam.asc
         sudo apt update
         sudo apt install chaste-dependencies
 

--- a/.github/workflows/ubuntu-tests.yml
+++ b/.github/workflows/ubuntu-tests.yml
@@ -46,7 +46,7 @@ jobs:
     - name: install dependencies
       run: |
         echo 'deb [signed-by=/usr/share/keyrings/chaste.asc] https://chaste.github.io/ubuntu ${{ matrix.codename }}/' | sudo tee -a /etc/apt/sources.list.d/chaste.list
-        sudo wget -O /usr/share/keyrings/chaste.asc https://chaste.github.io/ubuntu/ChasteTeam.asc
+        sudo wget -O /usr/share/keyrings/chaste.asc https://chaste.github.io/chaste.asc
         sudo apt update
         sudo apt install chaste-dependencies
 

